README.md

```diff
@@ -35,6 +35,7 @@ Creating this web application will provide exposure to:
 Model -
```

Comment:
Avoid vague statements like `Code is well organized into a meaningful file structure`. Also, be explicit about how you are going to achieve some of there, like `Functions are small and serve a single purpose`: Are you going to write separate tests for each function? Be explicit.